src/EditorFeatures/Core.Wpf/InheritanceMargin/InheritanceGlyphFactoryProvider.cs

```diff
@@ -16,18 +18,28 @@ namespace Microsoft.CodeAnalysis.Editor.InheritanceMargin
     [ContentType(ContentTypeNames.CSharpContentType)]
     [ContentType(ContentTypeNames.VisualBasicContentType)]
     [TagType(typeof(InheritanceMarginTag))]
-    [Order]
+    [Order(After = "VsTextMarker")]
```

Comment:
why?  please doc.

Reply:
I am testing it, and now I feel it should be after BreakpointGlyph, I will add comment